modules/ROOT/pages/mock-event-processor.adoc

```diff
@@ -165,6 +165,18 @@ The Mock When processor also enables you to mock errors specifying the cause (ex
 </munit-tools:mock-when>
 ----
 
+==== Parameters
+
+If you create a mocking error with exceptions, the following two parameters provide the trigger for the error in the *Then-return* tab:
+
+- *TypeId*: It follows a Mule Error type definition. For example, Namespace: Identifier.
```

Comment:
The example here is not actually an example. If you want an example of TypeId you can use the one that is mentioned previously in this section

Reply:
Thanks for the clarification!